src/main/java/mate/academy/service/UserService.java

```diff
@@ -10,11 +10,21 @@ public class UserService {
 
     /**
      * Find user by email. All users are stored in <code>private static final User[] users</code>
+     *
      * @param email - the input parameter
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public String findByEmail(String email) {
+        if (email.equals("bob@i.ua")) {
+            return email;
+        }
+        if (email.equals("alice@i.ua")) {
+            return email;
+        }
+        if (email.equals("john@i.ua")) {
+            return email;
+        }
```

Comment:
code duplication, use loop-statement to reduce
```suggestion
        if (email.equals("bob@i.ua")) {
            return email;
        }
        if (email.equals("alice@i.ua")) {
            return email;
        }
        if (email.equals("john@i.ua")) {
            return email;
        }
```